src/components/FormFields/styles.module.scss

```
@@ -1,115 +1,167 @@
+.item :global(.ant-form-item-explain-error) {
+    margin-top: 4px;
+    padding: 0 16px;
+    color: var(--form-error);
+    font-family: var(--m3-body-font-family);
+    font-size: 12px;
+    font-weight: 400;
+    letter-spacing: 0.4px;
+    line-height: 16px;
+}
+
+.item:global(.ant-form-item-has-error):not(:global(.ant-input-disabled)) .input,
```

Stylelint (17.13.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)